File: firebird/jython/delete/firebird_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	firebird_delete.py
#					Aug/28/2015
#
# ----------------------------------------------------------------
#
import sys
import	string
from java.lang import	System
#
import java
from java.sql import	DriverManager
#
# ----------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import delete_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")
id_in = sys.argv[1]
print ("%s" % id_in)
#
java.lang.Class.forName("org.firebirdsql.jdbc.FBDriver")
#
user = "sysdba"
password = "tiger"
database = "/var/tmp/firebird/cities.fdb"

url="jdbc:firebirdsql:host_firebird:" + database

conn = DriverManager.getConnection (url,user, password)
delete_proc	(conn,id_in)
#
conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------

File: firebird/jruby/delete/firebird_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	firebird_delete.rb
#
#					Aug/25/2015
#
# ------------------------------------------------------------
include Java
import org.firebirdsql.jdbc.FBDriver
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
key_in = ARGV[0]
#
puts key_in
#
user = "sysdba"
passwd = "tiger"
database = "/var/tmp/firebird/cities.fdb"

str_connect = "jdbc:firebirdsql:host_firebird:" + database

conn = DriverManager.getConnection(str_connect,user,passwd)
#
sql_delete_proc(conn,key_in)

conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------
